actions/views/create_comment.jsx

```diff
@@ -111,15 +111,15 @@ export function submitCommand(channelId, rootId, draft) {
 
         const {message} = draft;
 
-        return new Promise((resolve, reject) => {
-            ChannelActions.executeCommand(message, args, resolve, (err) => {
-                if (err.sendMessage) {
-                    dispatch(submitPost(channelId, rootId, draft));
-                } else {
-                    reject(err);
-                }
-            });
-        });
+        const {error} = await dispatch(executeCommand(message, args));
+
+        if (error) {
+            if (error.sendMessage) {
+                dispatch(submitPost(channelId, rootId, draft));
+            } else {
+                throw (error);
```

Comment:
why is this throwing an error instead of returning `{error}`

Reply:
The previous action was rejecting the promise, this does the same by throwing. I'm not looking to refactor all these actions in this PR.